### DISCUSSION TITLE
Module alias for imports in `main.ts` file

### DISCUSSION THREAD

**Discussion by ernestostifano** [storybookjs/storybook#22851] [Help]
Hi guys, I'm facing an issue when trying to import a package of my monorepo inside the `main.ts` file:

```ts
import {patchWebpackConfig} from '@my-monorepo/some-package';
import type {StorybookConfig} from '@storybook/react-webpack5';

const config: StorybookConfig = {
    // OTHER OPTIONS

    webpackFinal: (config, options) => {
        patchWebpackConfig(config, options.configType);

        return config;
    }

    // OTHER OPTIONS
};

export default config;
```

The problem is that, by default, this only works when the `@my-monorepo/some-package` package is built.

(`ERR! Error: Qualified path resolution failed: we looked for the following paths, but none could be accessed.`)

Now, this might seem a dumb question, but the whole purpose of that function is to make Storybook HMR work when the internal packages of the monorepo are not built, and it works! (by using Webpack aliases to redirect imports to the source code of the imported packages).

But, obviously, the `main.ts` file is not processed in the same way as the other files (manager/preview), so is there a way to pass an option to whatever tool you are using to process the `main.ts` file so I can tell it where to find the source code of the imported package?

For example, if you are using TypeScript, I could use `compilerOptions.paths`.

Because this is forcing me to build the packages each time I want to start Storybook, only for that function. Everything else works perfectly without building anything (because of Webpack aliases).

Thanks!

**Comment by darlesson** [comment]
@ernestostifano, would this be resolved by adding `".storybook/**/*"` under `include` in you TSConfig file?
